### ISSUE TITLE
CHANGELOG missing

### ISSUE THREAD

**Issue by MichaelDeBoey** [zestedesavoir/zmarkdown#381]
I just saw that `remark-numbered-footnotes` is updated to `v2.0.0`, but I can't find any CHANGELOG to check out the changes.

Would you guys be so kind to provide us with a decent CHANGELOG please? 🙂

Thanks for all your hard work! 🙂

**Comment by TitiAlone** [comment]
Hello,

unfortunately, this is a bit complicated for us to maintain a decent changelof because we maintain a lot of packages, but I'll try to do something in the upcoming releases.

But to answer your specific question, we recently released quite a lot of major versions (including `remark-numbered-footnotes`), because we updated all the packages' dependencies.

For `remark-numbered-footnotes`, special care is to be given to a new problem that cannot be solved (in the package) for now: the order of the footnotes doesn't match the references numbers, which can be quite annoying. We managed a workaround, but it involves revisiting the MDAST tree after parsing.

Anyway, all of this is explained in the [new README](https://github.com/zestedesavoir/zmarkdown/blob/master/packages/remark-numbered-footnotes/README.md) of the package, which should be read carefully before updating.

Feel free to ask if you have any more questions.
